sklearn/covariance/elliptic_envelope.py

```diff
@@ -68,25 +68,6 @@ class EllipticEnvelope(MinCovDet, OutlierMixin):
         such a way we obtain the expected number of outliers (samples with
         decision function < 0) in training.
 
-    Examples
-    --------
-    >>> import numpy as np
-    >>> from sklearn.covariance import EllipticEnvelope
-    >>> real_cov = np.array([[.8, .3],
-    ...                      [.3, .4]])
-    >>> np.random.seed(0)
-    >>> X = np.random.multivariate_normal(mean=[0, 0],
-    ...                                   cov=real_cov,
-    ...                                   size=300)
```

Comment:
You put 300 here, whereas it should be 500 if you want to have the same output as for MinCovDet